scripts/release_notes.rb

```diff
@@ -17,15 +17,37 @@
 # Create new release via API call
 # Create new search by open + release field
 
-unless @current_release_id = ARGV[0]
-  puts "Usage: #{$0} RELEASE_ID"
-  puts "Hint: 1.16.0 is 240"
+unless ARGV.size == 2
+  puts "Usage: #{$0} PROJECT RELEASE_NAME"
+  puts "Example: #{$0} foreman 1.18.0"
+  exit 1
+end
+
+@project_name = ARGV[0]
+@current_release_name = ARGV[1]
+
+# Determine version
+def get_version_id
+  url = "#{URL}/projects/#{@project_name}/versions.json"
+  uri = URI(URI.escape(url))
+  response = Net::HTTP.get(uri)
+	result = JSON.parse(response)
+  result['versions'].detect{|v| v['name'] == @current_release_name }['id']
```

Comment:
I think it's cleaner to check if the `result.detect` is nil than catching it. `rescue` could then be used to catch network errors.

Is there pagination in the versions?

Reply:
Not that I can see, no. You're right about the rescue, will fix tomorrow